content/implementations/forest.md

```diff
@@ -10,8 +10,20 @@ implRepos:
 
 # Forest
 
-Forest is an implementation of Filecoin written in Rust. The implementation will take a modular approach to building a full Filecoin node in two parts — (i) building Filecoin’s security critical systems in Rust from the Filecoin Protocol Specification, specifically the virtual machine, blockchain, and node system, and (ii) integrating functional components for storage mining and storage & retrieval markets to compose a fully functional Filecoin node implementation.
+Forest is an implementation of Filecoin written in Rust. It focuses on performance and low resource usage. It is compatible with most of the JSON-RPC API exposed by Lotus, making it easy to use with existing tools and libraries.
 
-You can find the Forest codebase [here](https://github.com/ChainSafe/forest) and the documentation site [here](https://chainsafe.github.io/forest/).
+Forest does not support all features of the reference implementation but is a good fit for specific applications, such as:
 
-The Forest implementation of Filecoin is supported by [ChainSafe](https://chainsafe.io/).
+- generating Filecoin snapshots,
+- running a bootstrap node,
+- running an RPC node.
+
+Forest does not support mining.
```

Comment:
I would reframe this as Forest is only a syncing implementation or Forest does not provide storage provider functionality. Just like we don't expect the Lotus node to do mining, that is the job lotus-miner etc.

Reply:
Rephrased.